src/Tizen.NUI/src/public/Position.cs

```diff
@@ -563,10 +569,13 @@ public float Y
         /// The z component.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        [Obsolete("Please do not use this setter, Deprecated in API8, will be removed in API10. please use new Position(...) constructor")]
         public float Z
         {
             set
             {
+                Tizen.Log.Fatal("NUI", "Please do not use this setter, Deprecated in API8, will be removed in API10. please use new Position(...) constructor");
```

Comment:
```suggestion
                Tizen.Log.Fatal("NUI", "Please do not use this setter, which is deprecated in API8 and will be removed in API10. Please use the new Position(...) constructor.");
```

Reply:
Thank you for review.
these are fixed by the following patches.

https://github.com/Samsung/TizenFX/pull/4356
https://github.com/Samsung/TizenFX/pull/4357

thank you.